fairseq/tasks/translation.py

```diff
@@ -92,6 +93,18 @@ def __init__(self, args, src_dict, tgt_dict):
         self.src_dict = src_dict
         self.tgt_dict = tgt_dict
 
+    @classmethod
+    def load_dictionary(cls, filename):
+        return Dictionary.load(filename)
+
+    @classmethod
+    def build_dictionary(cls, filenames, workers=1, threshold=-1, nwords=-1, padding_factor=8):
+        d = dictionary.Dictionary()
+        for filename in filenames:
+            Tokenizer.add_file_to_dictionary(filename, d, tokenizer.tokenize_line, workers)
+        d.finalize(threshold=threshold, nwords=nwords, padding_factor=padding_factor)
+        return d
```

Comment:
Let's move these implementations into the base class for now, so that LanguageModelTask (and some other internal Task variants) will continue to work.

Reply:
Done with commit https://github.com/pytorch/fairseq/pull/448/commits/03f0f3008a4dade4fd2330d9b34eb9023139a072